jake/iq/iq.py

```diff
@@ -136,6 +137,7 @@ def __handle_response(self, response: str) -> (bool):
       res = json.loads(response)
       LOG.debug(res)
       if res['policyAction'] == 'None':
+        self._policy_action = res['policyAction']
```

Comment:
minor: seems like `self._policy_action = res['policyAction']` always happens regardless of the `res['policyAction']` value, so maybe it should be set outside the `if / else`. e.g.

```
      self._policy_action = res['policyAction']
      if res['policyAction'] == 'None':
        LOG.debug("No policy issues, whew!")
      self._report_url = res['reportHtmlUrl']
```

Reply:
How this? 3c1a35c

Reply:
PERFECTO!

Reply:
CAPS LOCK FRIDAY IN PR - FOREVER